common/src/main/java/org/vivecraft/render/VRPlayerRenderer.java

```diff
@@ -76,17 +76,6 @@ public VRPlayerRenderer(EntityRendererProvider.Context p_174557_, boolean p_1745
 
     public void render(AbstractClientPlayer entityIn, float pEntityYaw, float pPartialTicks, PoseStack matrixStackIn, MultiBufferSource pBuffer, int pPackedLight)
     {
-        if (ClientDataHolder.getInstance().currentPass == RenderPass.GUI && entityIn.isLocalPlayer())
-        {
-            Matrix4f matrix4f = matrixStackIn.last().pose();
-            double d0 = (new Vec3((double)matrix4f.m00, (double)matrix4f.m01, (double)matrix4f.m02)).length();
-            matrixStackIn.last().pose().setIdentity();
-            matrixStackIn.last().normal().setIdentity();
-            matrixStackIn.translate(0.0D, 0.0D, 1000.0D);
-            matrixStackIn.scale((float)d0, (float)d0, (float)d0);
-            matrixStackIn.mulPose(Vector3f.ZP.rotationDegrees(180.0F));
-            matrixStackIn.mulPose(Vector3f.YP.rotationDegrees(180.0F + ClientDataHolder.getInstance().vrPlayer.vrdata_world_pre.getBodyYaw()));
-        }
```

Comment:
This is probably also gone with the PlayerRenderer Rewrite? But why is this not needed, it's gui rendering of the player?

Reply:
the renderer rewrite doesn't touch that part.
this is only for rendering the player in the gui. the stack reset messes with mods trying to render the player in the gui though
this does remove the fixed player int he gui though, if you want that, it now moves with the mouse cursor, like in vanilla